message_ix_models/model/water/report.py

```diff
@@ -1113,7 +1113,7 @@ def report(sc: Scenario, reg: str, sdgs: bool = False) -> None:
     for ur in ["urban", "rural"]:
         # CHANGE TO URBAN AND RURAL POP
         pop_tot = sc.timeseries(variable=("Population|" + ur.capitalize()))
-        pop_tot = pop_tot[-(pop_tot.region == "GLB region (R11)")]
+        pop_tot = pop_tot[-(pop_tot.region == "GLB region (R11)")]  # ONLY R11!!!
```

Comment:
What does this comment mean? Please record a longer version that enables others to understand it, too :)